### ISSUE TITLE
Apple Pencil Support

### ISSUE THREAD

**Issue by msztech** [Awalz/SwiftyDraw#20]
I'm considering adding a property that, if set, will only draw if the screen is touched with Apple Pencil. (It would be as easy as checking the source of the touch)

Also, we could detect the event from Apple Pencil (double tap) to switch the tools currently being used.

What do you think of the idea? I can work on this.

**Comment by LinusGeffarth** [comment]
Hey there, I'm sorry for the delayed response, I somehow missed your question.

I've actually implemented this in an app of mine using `shouldBeginDrawingIn` as I felt like this would be out of scope.  
We can implement it anyway, though. I'll just share my code...

Regarding the Pencil events, is that a feature of the new pencil? If you'd like, go for it and I'll happily merge it :)

**Comment by LinusGeffarth** [comment]
@msztech

**Comment by msztech** [comment]
Yes. It’s a new feature to switch between drawing and erasing. I’ll work on it and PR ;-)

**Comment by LinusGeffarth** [comment]
That's cool. Please check the release I just pushed (2.1.4) before starting to work on it.
I've implemented the finger / pencil logic.

You can now do:

```swift
drawView.allowedTouchTypes = [.finger, .pencil]
```

Leave out the one you'd like to exclude :)

**Comment by msztech** [comment]
@LinusGeffarth Pull Requested #22  :)
